src/com/snake/and/ladder/Game.java

```diff
@@ -0,0 +1,115 @@
+package com.snake.and.ladder;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Scanner;
+
+public class Game {
+
+  private List<Snake> snake;
```

Comment:
snake, ladder and person also denotes singular things whereas a list denotes plurality. These should be named snakes, ladders, persons.